{{cookiecutter.app_name}}/{{cookiecutter.app_name}}/user/models.py

```diff
@@ -9,48 +9,52 @@
 from ..extensions import encr_key, redis_store
 
 
-class Organization(SurrogatePK, Model):
-    __tablename__ = 'organization'
+class Merchant(SurrogatePK, Model):
+    __tablename__ = 'merchant'
 
     name = Column(db.String(80), nullable=False)
+    company_fid = Column(db.Integer, nullable=False)
     subdomain = Column(db.String(80), nullable=False, unique=True)
+    timezone = Column(db.String(80), nullable=False, unique=True)
     token = Column(EncryptedType(db.String, encr_key), nullable=False)
 
     @cached_property
     def fulfil_client(self):
         key = 'fulfil:user:%s:context' % self.token
         context = redis_store.get(key)
         if context:
-            client = Client(self.subdomain, auth=BearerAuth(self.token),
-                            context=json.loads(context))
+            client = Client(
+                self.subdomain, auth=BearerAuth(self.token),
+                context=json.loads(context)
+            )
         else:
             client = Client(self.subdomain, auth=BearerAuth(self.token))
             redis_store.set(key, json.dumps(client.context), 300)  # TTL 5min
         return client
 
     @classmethod
     def get_by_subdomain(cls, name):
-        """Find qrganization by subdomain
+        """Find merchant by subdomain
         """
         return cls.query.filter_by(subdomain=name).first()
 
     @classmethod
     def get_or_404(cls, subdomain):
-        organization = cls.get_by_subdomain(subdomain)
-        if not organization:
+        merchant = cls.get_by_subdomain(subdomain)
+        if not merchant:
             abort(404)
-        return organization
+        return merchant
 
     @classmethod
     def create_or_update(cls, subdomain, token, name):
-        organization = cls.get_by_subdomain(subdomain)
-        if not organization:
-            organization = cls()
-        organization.name = name
-        organization.subdomain = subdomain
-        organization.token = token
-        organization.save()
-        return organization
+        merchant = cls.get_by_subdomain(subdomain)
+        if not merchant:
+            merchant = cls()
+        merchant.name = name
+        merchant.subdomain = subdomain
+        merchant.token = token
+        merchant.save()
+        return merchant
 
 
 class User:
```

Comment:
Where is this class being used?

Reply:
in extensions.py https://github.com/fulfilio/cookiecutter-fulfil/pull/1/files#diff-950097abae480635b3beb2ea5778ebc5R57